shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -2180,7 +2180,7 @@ private SemanticsNode hitTest(float[] point) {
           return result;
         }
       }
-      return this;
+      return isFocusable() ? this : null;
```

Comment:
In iOS, we ask a11y service to ignore the node entirely by setting isAccessibilityElement = NO when the condition is met.
So I "think" this is ok

Reply:
Where do you see this in the iOS bridge? I'm looking at https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm#L233 and don't see focusability there. We use a few other things theree that we're not using here though.

Reply:
However, we do use most of the other logic from isFocusable in the iOS version

Reply:
yes, I mean iOS uses most of other logic from isFocusable except the addtional IS_FOCUSABLE flag. but i think we are trying to achieve the same thing in iOS. Except the isAccessibilityElement =NO is more stricter than AccessibilityNodeInfo.setFocusable(false) that the former will not absorb hit test, and i think we want to same behavior here?

Reply:
If not, we can go with implementing changes in framework as I mentioned in the PR description.

Reply:
I think this is probably fine. Let's just add a test to cover it.

Reply:
ok :) will do

Reply:
I think the sliver app bar does not overlay with the sliverlist, so nothing should be behind the appbar. I can't think of a use case where user might want to block something using non-focusable semantics node. It is already weird that if you have overlapping UI components.